libraries/AP_Motors/AP_MotorsMatrix.h

```diff
@@ -86,6 +86,23 @@ class AP_MotorsMatrix : public AP_MotorsMulticopter {
     // add_motor using raw roll, pitch, throttle and yaw factors
     void                add_motor_raw(int8_t motor_num, float roll_fac, float pitch_fac, float yaw_fac, uint8_t testing_order, float throttle_factor = 1.0f);
 
+    // structure to statically store motor information.  Entries match
+    // the arguments to add_motor - with the exception of the type,
+    // for compactness.
+    struct MotorDefInt {
+        int8_t motor_num;
+        int16_t angle_degrees;
+        int8_t yaw_factor;
+        uint8_t testing_order;
+    };
+    // same structure, but with floats
+    struct MotorDef {
```

Comment:
This looks like the MotorDef structure isn't used..

Reply:
Removed it